### ISSUE TITLE
Limit Request: catboost, catboost-dev - 80MB

### ISSUE THREAD

**Issue by dbakshee** [pypa/pypi-support#69]
**Please increase the file size limit from 60MB to 80MB.** 

For project [catboost-dev](https://pypi.org/project/catboost-dev) we have hit the size limit in a recent upload, and I suspect we'll hit it for the production project [catboost](https://pypi.org/project/catboost) in a coming release.

Typical size of catboost release is about 550MB, Release contains whl files for 3 OSes times python versions 2.7, 3.5, 3.6, 3.7. We also publish on conda.

Please increase index for PyPI. We do not use Test PyPI.

CatBoost is a machine learning library by Yandex, specifically it is gradient boosting on decision trees library. Its close alternatives/rivals are [LightGBM](https://pypi.org/project/lightgbm/) by Microsoft and [XGBoost](https://pypi.org/project/xgboost). From release to release the library steadily increased capabilities and now approached the limit of 60MB/file. We are going to add new functionality that will certainly increase the size and exceed the current limit.

**Comment by dbakshee** [comment]
Hi @di, as an update to the ticket, catboost team asks for a more generous file size limit and possibly no limit at all. Thanks, 
Dima.

**Comment by annaveronika** [comment]
Hi @di, could you please increase the limit? We want to publish the next release of CatBoost, and are blocked by this issue.

**Comment by di** [comment]
I've set the upload limit for `catboost` and `catboost-dev` to 100 MB on PyPI and TestPyPI.

As a warning: it looks like you're making nightly releases of `catboost-dev`. This is fine for now, but we'd be unlikely to increase the limit much more for this project due to the frequency of releases.

**Comment by di** [comment]
(Sorry, just on PyPI)

**Comment by dbakshee** [comment]
Thank you very much, this helped us a lot.

**Comment by di** [comment]
Actually, looking at https://pypi.org/stats/, the `catboost-dev` project is the 7th largest project on PyPI by total file size, with 149.5 GB in release files. Please consider switching to a less frequent release cycle for this project if possible.

**Comment by dbakshee** [comment]
Temporarily switched catboost_dev to once a week. Perhaps we'll move it to testpypi.

**Comment by di** [comment]
We have a long-term plan to shut down TestPyPI (https://github.com/pypa/warehouse/issues/726, https://github.com/pypa/warehouse/issues/918) so it probably makes the most sense to leave it on PyPI and keep the weekly cadence. (It's roughly the same cost to us regardless of where it's uploaded).
